RELEASING.md

```diff
@@ -0,0 +1,46 @@
+# Releasing
+
+1. Checkout your fork and make sure it's up-to-date with the `upstream`
+   ```console
+   $ git remote -v
+   origin     git@github.com:<your account>/tracee.git (fetch)
+   origin     git@github.com:<your account>/tracee.git (push)
+   upstream   git@github.com:aquasecurity/tracee.git (fetch)
+   upstream   git@github.com:aquasecurity/tracee.git (push)
+   ```
+   ```
+   git pull -r
+   git fetch upstream
+   git merge upstream/main
+   git push
+   ```
+1. Prepare release by creating the PR with the following changes
```

Comment:
I'm wondering if we want to keep using the latest tag in K8s deployment descriptors here: https://github.com/aquasecurity/tracee/blob/v0.7.0-rc-1/deploy/kubernetes/tracee-postee/tracee.yaml#L20

If there is a non backward compatible change in tracee image and we don't not update YAMLs it will fail in a way that's hard to troubleshoot without going through changes log.

Reply:
Using latest tag works fine with the latest release of tracee. However, if we we introduce new environment variable that must be set in K8s container image or extra volume mount, previous versions will stop working. That's why I'm usually pinning versions in K8s deployment descriptors. This also helps troubleshooting bugs when you know which exact version was running in a cluster. Last but not least, using the latest tag is not considered as good practice and many tools (including our own products) will warn about that.

Reply:
I added a note about this.